File: jdroid-java-webapp-core/src/main/java/com/jdroid/javaweb/application/AbstractAppModule.java
package com.jdroid.javaweb.application;

import java.util.Map;

public class AbstractAppModule implements AppModule {

	@Override
	public Map<String, String> getServerInfoMap() {
		return null;
	}

	@Override
	public void onCreateApplication() {
		// Do Nothing
	}

	@Override
	public void onContextDestroyed() {
		// Do Nothing
	}
}

File: jdroid-java-webapp-core/src/main/java/com/jdroid/javaweb/application/AbstractAppModule.kt
package com.jdroid.javaweb.application

open class AbstractAppModule : AppModule {

	override fun getServerInfoMap(): Map<String, String>? {
		return null
	}

	override fun onCreateApplication() {
		// Do Nothing
	}

	override fun onContextDestroyed() {
		// Do Nothing
	}
}
